### PR TITLE
Add the "Subscriptions" column display

### PR DESCRIPTION
**Pull request by wvega**
# Summary

This PR updates the Subscriptions integration to show a list of subscription orders next to framework tokens that are associated with one or more subscriptions.

## Details

There are some methods and filters that we will no longer use. Those will be deprecated [CH 30224](https://app.clubhouse.io/skyverge/story/30224/deprecate-unused-filter-handlers-in-subscriptions-integration).

## UI Changes

The Subscriptions column is visible in WooCommerce's payment methods table: https://cloud.skyver.ge/Z4u5J5nD

### Setup

- Configure NETBilling to use `dev-` version of the plugin framework (this branch)
- Activate Subscriptions and create a simple subscription product

### Purchase a subscription with a new credit card

1. Add the subscription product to the cart and go to the Checkout page
1. Purchase the subscription using a new credit card and make sure to save the payment method to your account
1. Go to MyAccount > Payment Methods
    - [x] The payment method is shown
    - [x] The Subscriptions column is shown
    - [x] There is a link to the subscription order

### Purchase a subscription with a saved payment method

1. Add the subscription product to the cart and go to the Checkout page
1. Purchase the subscription using the saved payment method
1. Go to MyAccount > Payment Methods
    - [ ] The payment method is shown
    - [ ] The Subscriptions column is shown
    - [ ] There are links to two subscription orders

### CODE REVIEW

File: woocommerce/payment-gateway/integrations/class-sv-wc-payment-gateway-integration-subscriptions.php [hunk 1]
```diff
@@ -661,6 +663,61 @@ public function add_my_payment_methods_table_header( $headers, $handler ) {
...
 
 
+	/**
+	 * Displays a list of subscriptions orders associated with the current token.
+	 *
+	 * This method only generates output for tokens created by the framework.
+	 *
+	 * TODO: Ask Subscriptions developers whether they would be interested in start adding the column to the payment methods table themselves {WV 2020-02-19}
+	 *
+	 * @since x.y.z
```
Comment (deivamagalhaes) [inline woocommerce/payment-gateway/integrations/class-sv-wc-payment-gateway-integration-subscriptions.php]: I have been using 5.6.0-dev. I just want to make sure we are using the same...

File: woocommerce/payment-gateway/integrations/class-sv-wc-payment-gateway-integration-subscriptions.php [hunk 2]
```diff
@@ -661,6 +663,61 @@ public function add_my_payment_methods_table_header( $headers, $handler ) {
...
+				echo $this->get_payment_method_subscriptions_html( $token );
+			}
+		}
+	}
+
+
+	/**
+	 * Gets the HTML code for the list of subscriptions orders associated with the given token.
+	 *
+	 * @since x.y.z
```
Comment (deivamagalhaes) [inline woocommerce/payment-gateway/integrations/class-sv-wc-payment-gateway-integration-subscriptions.php]: Same about the version.

File: woocommerce/payment-gateway/integrations/class-sv-wc-payment-gateway-integration-subscriptions.php [hunk 3]
```diff
@@ -661,6 +663,61 @@ public function add_my_payment_methods_table_header( $headers, $handler ) {
 	}
 
 
+	/**
+	 * Displays a list of subscriptions orders associated with the current token.
+	 *
+	 * This method only generates output for tokens created by the framework.
+	 *
+	 * TODO: Ask Subscriptions developers whether they would be interested in start adding the column to the payment methods table themselves {WV 2020-02-19}
+	 *
```
Comment (deivamagalhaes) [inline woocommerce/payment-gateway/integrations/class-sv-wc-payment-gateway-integration-subscriptions.php]: Missing @internal tag.

File: woocommerce/payment-gateway/integrations/class-sv-wc-payment-gateway-integration-subscriptions.php (hunk 1 above)
Comment (wvega) [reply woocommerce/payment-gateway/integrations/class-sv-wc-payment-gateway-integration-subscriptions.php]: Makes sense. I should have checked. I fixed the two instances here and two other I found in 318c777

File: woocommerce/payment-gateway/integrations/class-sv-wc-payment-gateway-integration-subscriptions.php (hunk 3 above)
Comment (wvega) [reply woocommerce/payment-gateway/integrations/class-sv-wc-payment-gateway-integration-subscriptions.php]: Thanks. Fixed in adc2326

**Pull request merged**